### ISSUE TITLE
mdt-gui crashed on start

### ISSUE THREAD

**Issue by kcgthb** [cbclab/MDT#4]
Hi there, 

I'm trying to start `mdt-gui`, but it crashes on start with the following Qt-related trace:

```
$ mdt-gui -d .
QObject::connect: Cannot connect QLineEdit::textChanged(QString) to (null)::directory_updated(QString)
Traceback (most recent call last):
  File "/share/software/user/open/py-mdt/0.10.6_py36/bin/mdt-gui", line 11, in <module>
    sys.exit(GUI.console_script())
  File "/share/software/user/open/py-mdt/0.10.6_py36/lib/python3.6/site-packages/mdt/shell_utils.py", line 44, in console_script
    cls().start(sys.argv[1:])
  File "/share/software/user/open/py-mdt/0.10.6_py36/lib/python3.6/site-packages/mdt/shell_utils.py", line 63, in start
    self.run(args, {})
  File "/share/software/user/open/py-mdt/0.10.6_py36/lib/python3.6/site-packages/mdt/cli_scripts/mdt_gui.py", line 40, in run
    start_gui(cwd)
  File "/share/software/user/open/py-mdt/0.10.6_py36/lib/python3.6/site-packages/mdt/gui/model_fit/qt_main.py", line 273, in start_gui
    composite_model_gui = MDTGUISingleModel(state, computations_thread)
  File "/share/software/user/open/py-mdt/0.10.6_py36/lib/python3.6/site-packages/mdt/gui/model_fit/qt_main.py", line 88, in __init__
    self.view_results_tab.setupUi(self.viewResultsTab)
  File "/share/software/user/open/py-mdt/0.10.6_py36/lib/python3.6/site-packages/mdt/gui/model_fit/tabs/view_results_tab.py", line 33, in setupUi
    self.selectedFolderText.textChanged.connect(self.directory_updated)
TypeError: connect() failed between textChanged(QString) and directory_updated()
```

Does that ring any bell? 

PyQt5 is installed, for Qt version 5.9.1.

Thanks!

**Comment by robbert-harms** [comment]
Hi there,

Thank you for your bug report. I was still using Qt version 5.6 which did not manifest this issue. After upgrading my system I could reproduce the issue and fix it. Please see if MDT version 0.10.7 also fixes this issue for you?

Best,

Robbert

**Comment by kcgthb** [comment]
Hi Robbert, 

Thanks for the quick fix, version 0.10.7 definitely fixed the problem!
